<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="3071" started-at="2021-01-29T06:43:28 IST" finished-at="2021-01-29T06:43:31 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="3071" started-at="2021-01-29T06:43:28 IST" finished-at="2021-01-29T06:43:31 IST">
      <class name="com.automation.PracticeSeleniumFramework.ABTest.VerifyABTest">
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:com.automation.PracticeSeleniumFramework.ABTest.VerifyABTest@10bbd20a]" name="launchBrowser" is-config="true" duration-ms="2211" started-at="2021-01-29T06:43:26 IST" finished-at="2021-01-29T06:43:28 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="launchApplication()[pri:0, instance:com.automation.PracticeSeleniumFramework.ABTest.VerifyABTest@10bbd20a]" name="launchApplication" is-config="true" duration-ms="2582" started-at="2021-01-29T06:43:28 IST" finished-at="2021-01-29T06:43:31 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method status="PASS" signature="initilazePages()[pri:0, instance:com.automation.PracticeSeleniumFramework.ABTest.VerifyABTest@10bbd20a]" name="initilazePages" is-config="true" duration-ms="27" started-at="2021-01-29T06:43:31 IST" finished-at="2021-01-29T06:43:31 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initilazePages -->
        <test-method status="FAIL" signature="verifyABTestLIink()[pri:0, instance:com.automation.PracticeSeleniumFramework.ABTest.VerifyABTest@10bbd20a]" name="verifyABTestLIink" duration-ms="431" started-at="2021-01-29T06:43:31 IST" finished-at="2021-01-29T06:43:31 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [A/B Test Control] but found [A/B Test Variant 1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [A/B Test Control] but found [A/B Test Variant 1]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.automation.PracticeSeleniumFramework.ABTest.VerifyABTest.verifyABTestLIink(VerifyABTest.java:34)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyABTestLIink -->
        <test-method status="PASS" signature="killSession()[pri:0, instance:com.automation.PracticeSeleniumFramework.ABTest.VerifyABTest@10bbd20a]" name="killSession" is-config="true" duration-ms="105" started-at="2021-01-29T06:43:36 IST" finished-at="2021-01-29T06:43:36 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- killSession -->
      </class> <!-- com.automation.PracticeSeleniumFramework.ABTest.VerifyABTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
